<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>KST_EXPORT</exportmacro>
 <class>VectorViewDialogWidget</class>
 <widget class="QWidget" name="VectorViewDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>664</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="GroupBox9_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Input Vectors</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="TextLabel1_6_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>X axis vector:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VectorSelector" name="_xVector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel1_6_2_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Y axis vector:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VectorSelector" name="_yVector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_2_2">
          <property name="text">
           <string>Interpolate to:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_interp">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="whatsThis">
           <string>Kst assumes that the first and last samples of the X &amp; Y axis vectors 
are pairs.  Consequently, if the X &amp; Y axis vectors have different numbers
of points, interpolation of one or the other will be required.  This combo-box set the policy for which vector will be used to set the resolution of the curve.</string>
          </property>
          <item>
           <property name="text">
            <string>Y Axis Vector</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>X Axis Vector</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High Resolution Vector</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low Resolution Vector</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>111</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>View Range</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout">
        <item row="1" column="1">
         <widget class="ScalarSelector" name="_yMinScalar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ScalarSelector" name="_xMinScalar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="_xMinCheckbox">
          <property name="text">
           <string>X Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="ScalarSelector" name="_xMaxScalar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="_yMinCheckbox">
          <property name="text">
           <string>Y Min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="_yMaxCheckbox">
          <property name="text">
           <string>Y Max</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="_xMaxCheckbox">
          <property name="text">
           <string>X Max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="ScalarSelector" name="_yMaxScalar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Set to:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_plotAxes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>32767</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <item>
           <property name="text">
            <string>XY Axes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>X Axis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y Axis</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>from plot:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_plotList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_realtimeButton">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text">
           <string>Realtime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_currentButton">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="text">
           <string>Current</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_3">
          <property name="text">
           <string>Flag Vector:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VectorSelector" name="_FlagVector" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer3_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="CurveAppearanceWidget" name="_curveAppearance" native="true"/>
   </item>
   <item>
    <widget class="CurvePlacementWidget" name="_curvePlacement" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
  <customwidget>
   <class>ScalarSelector</class>
   <extends>QWidget</extends>
   <header>scalarselector.h</header>
  </customwidget>
  <customwidget>
   <class>CurveAppearanceWidget</class>
   <extends>QWidget</extends>
   <header>curveappearancewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CurvePlacementWidget</class>
   <extends>QWidget</extends>
   <header>curveplacementwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_xVector</tabstop>
  <tabstop>_yVector</tabstop>
  <tabstop>_interp</tabstop>
  <tabstop>_xMinCheckbox</tabstop>
  <tabstop>_xMinScalar</tabstop>
  <tabstop>_xMaxCheckbox</tabstop>
  <tabstop>_xMaxScalar</tabstop>
  <tabstop>_yMinCheckbox</tabstop>
  <tabstop>_yMinScalar</tabstop>
  <tabstop>_yMaxCheckbox</tabstop>
  <tabstop>_yMaxScalar</tabstop>
  <tabstop>_plotAxes</tabstop>
  <tabstop>_plotList</tabstop>
  <tabstop>_realtimeButton</tabstop>
  <tabstop>_currentButton</tabstop>
  <tabstop>_FlagVector</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
  <include location="local">vectorselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">vectorselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">scalarselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">scalarselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">scalarselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">scalarselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">kcombobox.h</include>
  <include location="local">vectorselector.h</include>
  <include location="local">kstcombobox.h</include>
  <include location="local">curveappearancewidget.h</include>
  <include location="local">curveplacementwidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
